OpenTabletDriver.UI.Library/Controls/NavigationHost.axaml.cs

```diff
@@ -0,0 +1,104 @@
+using Avalonia;
+using Avalonia.Animation;
+using Avalonia.Controls;
+using CommunityToolkit.Mvvm.DependencyInjection;
+using OpenTabletDriver.UI.Navigation;
+
+namespace OpenTabletDriver.UI.Controls;
+
+// TODO: Convert to templated control
+
+/// <summary>
+/// Displays <see cref="ContentControl.Content"/> with different possible
+/// transitions for different navigation types.
+/// </summary>
+public partial class NavigationHost : UserControl
+{
+    private readonly INavigatorFactory _navigatorFactory;
+    private INavigator? _navigator;
+
+    public NavigationHost()
+    {
+        InitializeComponent();
+        _navigatorFactory = Ioc.Default.GetRequiredService<INavigatorFactory>();
+    }
+
+    public static readonly StyledProperty<string?> NavigationHostNameProperty =
+        AvaloniaProperty.Register<NavigationHost, string?>(nameof(NavigationHostName));
+
+    public string? NavigationHostName
+    {
+        get => GetValue(NavigationHostNameProperty);
+        set => SetValue(NavigationHostNameProperty, value);
+    }
+
+    public static readonly StyledProperty<IPageTransition?> NextTransitionProperty =
+        AvaloniaProperty.Register<NavigationHost, IPageTransition?>(nameof(NextTransition),
+            new CrossFade(TimeSpan.FromSeconds(0.125)));
```

Comment:
expose timespans in a configuration somewhere, e.g. Build.props

Reply:
Or an entry in `OpenTabletDriver.UI.Library/Styles`. Using `Dierctory.Build.props` here complicates things as that requires adding something similar to `OpenTabletDriverSettingsFormat` from above reviews.

This is a default value fwiw and conventionally we don't set things here. Should instead override this property where this class is constructed if we wanna follow conventions.